.gitbook/3-protocol-devs/1-dep-specs/2-data-deal.md

```diff
@@ -0,0 +1,117 @@
+# Data Deal
+
+- Status: Draft
+- Crated: 2023-01-03
+- Modified: 2023-01-03
+- Authors
+  - Youngjoon Lee <yjlee@medibloc.org>
+  - Gyuguen Jang <gyuguen.jang@medibloc.org>
+  - Hansol Lee <hansol@medibloc.org>
+  - Myongsik Gong <myongsik_gong@medibloc.org>
+  - Inchul Song <icsong@medibloc.org>
+
+
+## Synopsis
+
+This document defines a data deal which is a contract for data collecting and pay for provision in [DEP](../../1-users/3-data-exchange/0-about-dep.md).
+Data consumers create data deal by specifying what type of data and how much they want, and how much they are willing to pay for it.
+Data providers can provide their data to the deals that match the type of data they have.
+When a data provider provides data to the deal, the payout is distributed to the provider and oracle that validated the data.
```

Comment:
```suggestion
When a data provider provides data to the deal, the payout is distributed to the provider and an oracle that validated the data.
```

Reply:
I add `the` instead of `an` because it is being specified which oracle it is (which validated the data).